analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp6.cs

```diff
@@ -57,7 +57,7 @@ public void Repro_1324(object o)
             }
             catch (ApplicationException) when (o == null)
             {
-                var b = o.ToString(); // Noncompliant
+                var b = o.ToString(); // Non-compliant Suppressed #6117
```

Comment:
The `Non-compliant` is confusing on master. It was fine for the active WIP stage.
```suggestion
                var b = o.ToString(); // FN Suppressed #6117
```

Reply:
Same below